RELEASES.md

```diff
@@ -10,4060 +8,37 @@ Unreleased.
-  directly.
-  [#1363](https://github.com/bytecodealliance/wasmtime/pull/1363)
-
-* The Cranelift repository has been merged into the Wasmtime repository.
-
-* Support for interface types has been temporarily removed.
-  [#1292](https://github.com/bytecodealliance/wasmtime/pull/1292)
-
-* The exit code of the `wasmtime` CLI has changed if the program traps.
-  [#1274](https://github.com/bytecodealliance/wasmtime/pull/1274)
-
-* The `wasmtime` CLI now logs to stderr by default and the `-d` flag has been
-  renamed to `--log-to-file`.
-  [#1266](https://github.com/bytecodealliance/wasmtime/pull/1266)
-
-* Values cannot cross `Store` objects, meaning you can't instantiate a module
-  with values from different stores nor pass values from different stores into
-  methods.
-  [#1016](https://github.com/bytecodealliance/wasmtime/pull/1016)
-
---------------------------------------------------------------------------------
-
-## 0.12.0
-
-Released 2020-02-26.
-
-### Added
-
-* Support for the [WebAssembly text annotations proposal][annotations-proposal]
-  has been added.
-  [#998](https://github.com/bytecodealliance/wasmtime/pull/998)
-
-* An initial C API for instantiating WASI modules has been added.
-  [#977](https://github.com/bytecodealliance/wasmtime/pull/977)
-
-* A new suite of `Func::getN` functions have been added to the `wasmtime` API to
-  call statically-known function signatures in a highly optimized fashion.
-  [#955](https://github.com/bytecodealliance/wasmtime/pull/955)
-
-* Initial support for profiling JIT code through perf jitdump has been added.
-  [#360](https://github.com/bytecodealliance/wasmtime/pull/360)
-
-* More CLI flags corresponding to proposed WebAssembly features have been added.
-  [#917](https://github.com/bytecodealliance/wasmtime/pull/917)
-
-[annotations-proposal]: https://github.com/webassembly/annotations
-
-### Changed
-
-* The `wasmtime` CLI as well as embedding API will optimize WebAssembly code by
-  default now.
-  [#973](https://github.com/bytecodealliance/wasmtime/pull/973)
-  [#988](https://github.com/bytecodealliance/wasmtime/pull/988)
-
-* The `verifier` pass in Cranelift is now no longer run by default when using
-  the embedding API.
-  [#882](https://github.com/bytecodealliance/wasmtime/pull/882)
-
-### Fixed
-
-* Code caching now accurately accounts for optimization levels, ensuring that if
-  you ask for optimized code you're not accidentally handed unoptimized code
-  from the cache.
-  [#974](https://github.com/bytecodealliance/wasmtime/pull/974)
-
-* Automated releases for tags should be up and running again, along with
-  automatic publication of the `wasmtime` Python package.
-  [#971](https://github.com/bytecodealliance/wasmtime/pull/971)
+Release notes for previous releases of Wasmtime can be found on the respective
+release branches of the Wasmtime repository.
+
+<!-- ARCHIVE_START -->
+* [21.0.0](https://github.com/bytecodealliance/wasmtime/blob/release-21.0.0/RELEASES.md)
```

Comment:
I *think* we don't need to include point-releases, since there is no separate branch to link to (and the `release-X.0.0` branch will contain all updates from `X.Y.Z`); but should we note somewhere, e.g. list the point-releases here in the link text or after the link?

Otherwise I could imagine some confusion: I'm curious what was released in 21.0.1, I look at this list, I don't see the version number and get confused (or grep for it, same result).

Reply:
Alternately: name the link text something like "21.x releases" or "21.0.0 and following" or ...

Reply:
I actually originally had this written here as 21.0.x but I ended up renaming since it was nontrivial to create the string "21.0.x" from the string "21.0.0" available in the release scripts in github actions. I think you're right though, I'll try to switch this to "21.0.x" (along with everything below), to make it more clear that it's for all those releases.

Reply:
I also regret choosing the release branch name as `release-N.0.0` originally since that'd be more appropriate as `release-N.0.x` as well...